### DISCUSSION TITLE
Why is it increment/decrement under the cursor instead of next?

### DISCUSSION THREAD

**Discussion by txtyash** [helix-editor/helix#1199] [Q&A] [answered]
`ctrl-a` and `ctrl-x` don't act on objects not under the cursor whereas in vim they act on any next object.

Why is it not the vim way?

**Comment by archseer** [comment, marked as answer]
Read the discussion on the [implementation PR](https://github.com/helix-editor/helix/pull/1027#issuecomment-964791596)

**Reply by txtyash** [reply]
So is it going to be implemented or going to function the same way?

**Reply by pickfire** [reply]
> > Jumping to the first number if the cursor isn't already on a number
>
> Most of the time when I use ctrl-a/x in vim I rely on it jumping to the nearest number, so personally I do like this feature.

Based on what @archseer shared, he shared the link to current implementation done by @sudormrfbin which explains why it behaves that way. It's not a bug, it's a feature.
